.yamato/com.unity.ml-agents-test.yml

```diff
@@ -24,6 +30,17 @@ test_platforms:
     flavor: b1.medium
 ---
 
+all_package_tests:
```

Comment:
What's the purpose of this block? Looks like it waits for other tests, but doesn't do anything and nothing waits on it.

Reply:
i wanted to use it for the publish/promote tests. I separated out the trunk tests from the "every commit tests" because sometimes it still waits for katana to build Unity (which took 3 hours today).  New commit makes use of this idea.